File: fe/approval-engine/src/models/student.ts
export interface Student {
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  created: string;
  modified: string;
  status: string;
}

export interface CreateStudent {
  firstName: string;
  lastName: string;
  email: string;
}

export interface GetStudentsQuery {
  pageIndex: number;
  pageSize: number;
}

File: src/SampleApp.Core/Data/Entities/Student.cs
using SampleApp.Core.Enums;

namespace SampleApp.Core.Data.Entities
{
    public class Student
    {
        public long Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public DateTime Created { get; set; }
        public DateTime? Modified { get; set; }
        public StudentStatus Status { get; set; }
    }
}
